src/index.js

```diff
@@ -1,18 +1,9 @@
 import Exporter from './exporter';
 import createTemplate from './template';
 
-let sql;
-
-if (process.env.APP_ENV === 'browser' || typeof window !== 'undefined') {
-  require('script-loader!sql.js');
-  sql = window.SQL;
-} else {
-  sql = require('sql.js');
-}
```

Comment:
@yakhinvadim @jamsinclair what do you think? `sql.js` is now configurable so it makes sense to leave it up to users of the library (actually, it should be like this from the start - without hardcode to `webpack`'s script semantics :grimacing: )

Reply:
I'm not sure I understand, why do we need to exclude sql.js from this package? To allow users to load in asynchronously in the browser?
If yes, since anki-apkg-export doesn't work without sql.js maybe we should include sql.js in this package and let them load the entire package asynchronously?

Reply:
There are too many options now: `sql.js` comes with 2 distributions (old `asm.js` and new `wasm`) and we can bundle code via `browserify`, `webpack`, `rollup`, `parcel` or even plain `<script />` tag. While the old approach limits us to exactly one combination: `asm.js` - `webpack`

Reply:
Although I prefer opinionated tools, what you're saying makes sense. Let's leave it up to users.